networkx/algorithms/perfect_graph.py

```diff
@@ -0,0 +1,37 @@
+import networkx as nx
+
+__all__ = ["is_perfect_graph"]
+
+
+def is_perfect_graph(G):
+    """Return True if G is a perfect graph, else False.
+
+    According to the Strong Perfect Graph Theorem, a graph is perfect
+    if and only if neither it nor its complement contains an induced
+    odd cycle of length at least 5.
```

Comment:
Similarly this docstring could include some information about the time complexity of the algorithm, since this is one of the big payoffs of the strong perfect graph theorem (i.e. the existence of a polynomial-time algorithm for checking whether a graph if perfect).

Finally, the docstring should be formatted according to the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections) - i.e. there should be a `Parameters` section, etc. See other docstrings in the library for inspiration!